### PR TITLE
Fix tailog pages IDs

### LINKED ISSUE #1412: tailog.php/tailog-FTL.php duplicate IDs
Should be fixed before v5.1.0

```html
<input type="checkbox" name="active" id="active" checked id="chk1"><label for="active">Automatic scrolling on update</label>
<pre id="output" style="width: 100%; height: 100%; max-height:650px; overflow-y:scroll;"></pre>
<input type="checkbox" name="active" id="active" checked id="chk2"><label for="active">Automatic scrolling on update</label>
```

Also, we should double check the `name` attributes.

### PR DESCRIPTION
**Pull request by XhmikosR**
Signed-off-by: XhmikosR <xhmikosr@gmail.com>

**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

Fixes #1412

### CODE REVIEW

File: taillog-FTL.php [hunk 1]
```diff
@@ -12,9 +12,13 @@
     <h1>Output the last lines of the pihole-FTL.log file (live)</h1>
 </div>
 
-<input type="checkbox" name="active" id="active" checked id="chk1"><label for="active">Automatic scrolling on update</label>
+<input type="checkbox" checked id="chk1">
+<label for="chk1">Automatic scrolling on update</label>
```
Comment (DL6ER) [inline taillog-FTL.php:16]:
```suggestion
<div>
    <input type="checkbox" checked id="chk1">
    <label for="chk1">Automatic scrolling on update</label>
</div>
```

File: taillog-FTL.php [hunk 2]
```diff
@@ -12,9 +12,13 @@
...
+
 <pre id="output" style="width: 100%; height: 100%; max-height:650px; overflow-y:scroll;"></pre>
-<input type="checkbox" name="active" id="active" checked id="chk2"><label for="active">Automatic scrolling on update</label>
+
+<input type="checkbox" checked id="chk2">
+<label for="chk2">Automatic scrolling on update</label>
```
Comment (DL6ER) [inline taillog-FTL.php:21]:
```suggestion
<div>
    <input type="checkbox" checked id="chk2">
    <label for="chk2">Automatic scrolling on update</label>
</div>
```

File: taillog.php [hunk 3]
```diff
@@ -12,9 +12,13 @@
     <h1>Output the last lines of the pihole.log file (live)</h1>
 </div>
 
-<input type="checkbox" name="active" id="active" checked id="chk1"><label for="active">Automatic scrolling on update</label>
+<input type="checkbox" checked id="chk1">
+<label for="chk1">Automatic scrolling on update</label>
```
Comment (DL6ER) [inline taillog.php:16]:
```suggestion
<div>
    <input type="checkbox" checked id="chk1">
    <label for="chk1">Automatic scrolling on update</label>
</div>
```

File: taillog.php [hunk 4]
```diff
@@ -12,9 +12,13 @@
...
+
 <pre id="output" style="width: 100%; height: 100%; max-height:650px; overflow-y:scroll;"></pre>
-<input type="checkbox" name="active" id="active" checked id="chk2"><label for="active">Automatic scrolling on update</label>
+
+<input type="checkbox" checked id="chk2">
+<label for="chk2">Automatic scrolling on update</label>
```
Comment (DL6ER) [inline taillog.php:21]:
```suggestion
<div>
    <input type="checkbox" checked id="chk2">
    <label for="chk2">Automatic scrolling on update</label>
</div>
```

**Review by DL6ER — DISMISSED** [review]
We need the empty `<div>`s to get proper checkbox styling with `icheck-bootstrap`.

Your branch:
Screenshot from 2020-06-04 14-32-35


With empty `<div>` elements as suggested:
Screenshot from 2020-06-04 14-32-24

**Comment by XhmikosR** [comment]
As you can see the empty divs are not present in devel

**Comment by XhmikosR** [comment]
You are welcome to fix those separately.

**Comment by XhmikosR** [comment]
Also, these pages do not follow the basic AdminLTE markup, it's not a checkbox issue. They are missing more needed classes, but I don't want to fix them in devel since I fixed those in the AdminLTE 3 branch.

**Comment by DL6ER** [comment]
The check boxes are broken now. I agree. Fixing them along in this PR will not hurt and I don't see a justification for leaving them broken when changing them.

**Comment by XhmikosR** [comment]
I want to keep my PRs clean. This does one thing. You are welcome to fix the other issues on a separate PR.

**Review by DL6ER — APPROVED** [review]

**Pull request merged**